<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimalDialog</class>
 <widget class="QDialog" name="AnimalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="investigatorHorizontalLayout">
     <item>
      <widget class="QLabel" name="investigatorLabel">
       <property name="text">
        <string>Investigator: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="investigatorComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="asiHorizontalLayout">
     <item>
      <widget class="QLabel" name="asiLabel">
       <property name="text">
        <string>Animal Services ID: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="asiLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dobHorizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date of Birth: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sexHorizontalLayout">
     <item>
      <widget class="QLabel" name="sexLabel">
       <property name="text">
        <string>Sex: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="maleRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="femaleRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="unknownRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="genotypeHorizontalLayout">
     <item>
      <widget class="QLabel" name="genotypeLabel">
       <property name="text">
        <string>Genotype: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="genotypeLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nicknameHorizontalLayout">
     <item>
      <widget class="QLabel" name="nicknameLabel">
       <property name="text">
        <string>Nickname: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nicknameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="surgeryHorizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="surgeryVerticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Surgical Log: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Add ...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnimalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnimalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
